### ISSUE TITLE
Adding an instance with Superset for visualization

### ISSUE THREAD

**Issue by villasv** [villasv/turbine#29]
Should visualization become part of the default stack? Probably not if we think in separation of concerns, but it's pretty useful and possibly wanted by many people (me included).

**Comment by villasv** [comment]
This should be done only after there's a kind of columnar storage. And that will only happen after we have a suitable feature storage instead of Postgres.

**Comment by villasv** [comment]
Definitely does not belong here, even though it would be pretty awesome.

**Comment by villasv** [comment]
Changed my previous opinion of "definitely" to "perhaps". It's true that Superset is very general, but it's frequently used also as a monitoring tool for Airflow tasks.

**Comment by villasv** [comment]
Nope. It's really tempting, but it's out of scope indeed.
